cloud/jenkins/psmdb_operator_gke_latest.groovy

```diff
@@ -1,29 +1,88 @@
 region='us-central1-a'
 tests=[]
 clusters=[]
-
+release_params="source/e2e-tests/release_params"
 
 void verifyParams() {
-    if ("$RELEASE_RUN" == "YES" && (!"$PILLAR_VERSION" && !"$IMAGE_MONGOD")){
-        error("This is RELEASE_RUN. Either PILLAR_VERSION or IMAGE_MONGOD should be provided")
+    if ("$RELEASE_RUN" == "YES") {
+        echo "=========================[ RELEASE RUN ]========================="
+        if (!"$PILLAR_VERSION" && !"$IMAGE_MONGOD") {
+            error("Either PILLAR_VERSION or IMAGE_MONGOD should be provided for release run!")
+        }
+        if ("$GKE_RELEASE_CHANNEL" != "STABLE".toLowerCase()) {
```

Comment:
Can't we automate this? If we start release run - we use stable?

Reply:
done